cypress/tests/ui/experiment-tracking/panel.cy.js

```diff
@@ -3,7 +3,7 @@
 describe('Experiment Tracking Primary Toolbar', () => {
   it('verifies that users can hide/show the side menu. #TC-38', () => {
     // Alias
-    cy.get('[data-test="btnToggleMenu"]').as('btnToggleMenu');
+    cy.get('[data-test*="sidebar-experiments-visible-btn-"]').as('btnToggleMenu');
```

Comment:
Hey is there a special meaning for `*=`?

Reply:
@jitu5 -- see above. 

Maybe I need to document this somewhere.